src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
```

Comment:
Better add an additional container here

It is not a universal rule (and even more - it does not affect the task), but as long as the task sounds like implementing some design library element, it is better to have it inside a separate isolated container (just a good habit).